android/src/main/java/com/hypertrack/sdk/capacitor/HyperTrackSdkPlugin.java

```diff
@@ -58,4 +65,245 @@ public void enableDebugLogging(PluginCall call) {
             call.reject(e.toString(),e);
         }
     }
+
+    @PluginMethod
+    public void hyperTrackRequestPermissions(PluginCall call) {
+        implementation.print("hyperTrackRequestPermissions called");
+        try {
+            implementation.hyperTrackRequestPermissions();
+            call.resolve();
+        } catch (Exception e) {
+            call.reject(e.toString(),e);
+        }
+    }
+
+    @PluginMethod
+    public void hyperTrackAllowMockLocations(PluginCall call) {
+        implementation.print("hyperTrackAllowMockLocations called");
+        try {
+           implementation.hyperTrackAllowMockLocations();
+            call.resolve();
+        } catch (Exception e) {
+            call.reject(e.toString(),e);
+        }
+    }
+
+    @PluginMethod
+    public void getDeviceId(PluginCall call) {
+        implementation.print("getDeviceId called");
+        try {
+            String deviceId = implementation.getDeviceId();
+            JSObject ret = new JSObject();
+            ret.put("deviceId", implementation.print(deviceId));
+            call.resolve(ret);
+        } catch (Exception e) {
+            call.reject(e.toString(),e);
```

Comment:
formatting issue (please use autoformat)

Reply:
I am already using that. I think, I have to do change something config.......let me check that